### PR TITLE
2947 fix undefined routes

### PR DESCRIPTION
**Pull request by francisl**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -193,6 +193,37 @@ const App = compose(
...
+          path={ROUTES.studies}
+          exact
+          render={(props) =>
+            protectRoute({
+              api,
+              isLoadingUser,
+              Component: () => {
+                import('pages/studies')
+                  .then(({ StudiesPage }) => StudiesPage)
+                  .catch(() => {});
```
Comment (evans-g-crsj) [inline src/App.js:220]: is the "catch" necessary?

File: src/App.js [hunk 2]
```diff
@@ -193,6 +193,37 @@ const App = compose(
...
+              },
+              loggedInUser,
+              ...props,
```
Comment (evans-g-crsj) [inline src/App.js:223]: if possible, let's avoid the ...props

**Review by evans-g-crsj — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (francisl) [reply src/App.js:220]: I'll add handling the catch clause in next update

File: src/App.js (hunk 2 above)
Comment (francisl) [reply src/App.js:223]: in next update

File: src/App.js [hunk 3]
```diff
@@ -193,6 +193,37 @@ const App = compose(
...
+              api,
+              isLoadingUser,
+              Component: UserDashboard,
+              loggedInUser,
+              ...props,
+            })
+          }
+        />
+
+        <Route
```
Comment (evans-g-crsj) [inline src/App.js:210]: shouldn't that be wrapped in a feature toggle too?

**Review by evans-g-crsj — CHANGES_REQUESTED** [review]

**Review by evans-g-crsj — APPROVED** [review]

**Pull request merged**